### ISSUE TITLE
Questions about Training Resources

### ISSUE THREAD

**Issue by Amazingren** [IDKiro/gUNet#2]
Hi @IDKiro,

Thanks for your impressive work. Recently we are going to do some related work. Hence I am curious about the training resources issue.

May I kindly ask how many GPUs are used when training such a model? And What kind of GPUs? And how long it takes for the training?

Best regards,

**Comment by IDKiro** [comment]
RTX3090 * 4 for default.
You can adjust the batch size and learning rate in the config file according to the linear scaling rules, and SyncBN will be automatically enabled when necessary.
